content/en/docs/tasks/tools/_index.md

```diff
@@ -47,3 +47,13 @@ installed and configured.
 you need to do to get up and running with `kind`.
 
 <a class="btn btn-primary" href="https://kind.sigs.k8s.io/docs/user/quick-start/" role="button" aria-label="View kind Quick Start Guide">View kind Quick Start Guide</a>
+
+## kubeadm
+
+You can use the {{< glossary_tooltip term_id="kubeadm" text="kubeadm" >}} tool to create and manage Kubernetes clusters.
+It performs the actions necessary to get a minimum viable, secure cluster up and running in a user friendly way.
+
+[`Quick Install`](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) show you how to install `kubeadm`.
```

Comment:
I checked the title of the page this links to. I think it should be:
```suggestion
[Installing kubeadm](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) show you how to install `kubeadm`.
```

Reply:
please also change to:
> shows you how to install kubeadm.

- ^ no quotes around "kubeadm"
- show -> shows

Reply:
ok